ImplicitPackageReferenceUnitTests/ImplicitPackageReferenceUnitTests.cs

```diff
@@ -124,86 +148,55 @@ public void WhenTaskIsPassedXMLFileTaskReturnsFalse()
             var implicitPacker = new AddImplicitPackageReferences(log);
             TaskItem implicitDependency = new TaskItem("Microsoft.AspNetCore.Http.Abstractions");
             implicitPacker.AssetsFilePath = "WrongFile.xml";
-            implicitPacker.DependenciesToVersionAndPackage = new TaskItem[] { implicitDependency };
+            implicitPacker.ImplicitPackageReferences = new TaskItem[] { implicitDependency };
             Assert.IsFalse(implicitPacker.Execute());
             Assert.AreEqual("AddImplicitPackageReferences failed, could not parse file WrongFile.xml. Exception:Unexpected character encountered while parsing value: <. Path '', line 0, position 0.", log.ErrorMessage);
         }
 
-        [TestMethod]
-        [DeploymentItem(@"MissingLibrary.json")]
-        public void WhenJsonFileIsMissingLibrarySectionTaskReturnsFalse()
-        {
-            var implicitPacker = new AddImplicitPackageReferences(log);
-            TaskItem implicitDependency = new TaskItem("Microsoft.AspNetCore.Http.Abstractions");
-            implicitPacker.AssetsFilePath = "MissingLibrary.json";
-            implicitPacker.DependenciesToVersionAndPackage = new TaskItem[] { implicitDependency };
-
-            JObject assetsFile = JObject.Parse(File.ReadAllText("MissingLibrary.json"));
-            Assert.IsNull(assetsFile["project"]["frameworks"]["netstandard2.0"]["dependencies"]["Microsoft.AspNetCore.Http.Abstractions"]);
-
-            Assert.IsFalse(implicitPacker.Execute());
-            assetsFile = JObject.Parse(File.ReadAllText("MissingLibrary.json"));
-            Assert.IsNull(assetsFile["project"]["frameworks"]["netstandard2.0"]["dependencies"]["Microsoft.AspNetCore.Http.Abstractions"]);
-            Assert.AreEqual("AddImplicitPackageReferences failed, MissingLibrary.json missing Libraries section.", log.ErrorMessage);
-        }
-
-        [TestMethod]
-        [DeploymentItem("VersionlessPackage.json")]
-        public void WhenPackageIsMissingVersionNumberInJsonFileTaskReturnsFalse()
```

Comment:
This is also another test that makes no sense.  The project.assets.json would have to be really malformed for this to happen, and there are 1000 other possible malformations that might happen, no need to test to ensure they are all covered.